cumulus_library/study_parser.py

```diff
@@ -86,13 +86,25 @@ def get_study_prefix(self) -> Optional[str]:
         """
         return self._study_config.get("study_prefix")
 
-    def get_sql_file_list(self) -> Optional[StrList]:
+    def get_sql_file_list(self, continue_from: str = None) -> Optional[StrList]:
         """Reads the contents of the sql_config array from the manifest
 
         :returns: An array of sql files from the manifest, or None if not found.
         """
         sql_config = self._study_config.get("sql_config", {})
-        return sql_config.get("file_names", [])
+        sql_files = sql_config.get("file_names", [])
+        if continue_from:
+            match_found = False
+            for file in sql_files:
+                print(continue_from.replace(".sql", ""))
+                print(file.replace(".sql", ""))
+                if continue_from.replace(".sql", "") == file.replace(".sql", ""):
+                    sql_files = sql_files[sql_files.index(file) :]
```

Comment:
nit: you could save yourself an `index()` call by doing `for i, file in enumerate(sql_files):` above, but :shrug:

Reply:
ooh, i dig it